src/colors.yaml

```diff
@@ -4,12 +4,14 @@ base:
    comment: ""
    colors:
       white: &white [255,255,255,1]
+      whiteTransp05: &whiteTransp05 [255,255,255,0.05]
```

Comment:
Some possible conventions for this:
`transparent[Color][Light|Medium|Dark]`
`transparent[Color][Less|More]`
`transparent[Color][--1|--2|--3]`
`transparent[Color][Thin|Medium|Tick]` (thinking of opacity in a "layer" sense)